src/Compute/Compute/StorageServices/AddAzureVhdCommand.cs

```diff
@@ -25,51 +25,102 @@
 using System.IO;
 using System.Management.Automation;
 using Rsrc = Microsoft.Azure.Commands.Compute.Properties.Resources;
+using Microsoft.Azure.Management.Compute.Models;
+using Microsoft.Azure.Commands.Compute.Automation.Models;
+using Microsoft.WindowsAzure.Commands.Utilities.Common;
+using Microsoft.Azure.Commands.Compute.Automation;
+using Microsoft.Azure.Management.Compute;
+using Microsoft.Azure.Commands.Compute.Sync.Upload;
+using Microsoft.WindowsAzure.Commands.Sync;
+using System.Management;
+using Microsoft.Samples.HyperV.Storage;
+using Microsoft.Samples.HyperV.Common;
+using System.Threading;
 
 namespace Microsoft.Azure.Commands.Compute.StorageServices
 {
     /// <summary>
     /// Uploads a vhd as fixed disk format vhd to a blob in Microsoft Azure Storage
     /// </summary>
     [GenericBreakingChange("This cmdlet will now try to resize the VHD file to (N * Mib + 512 bytes) using Hyper-V before uploading.\n If Hyper-V is not found, user will have to either enabled Hyper-V or resize manually before uploading.")]
```

Comment:
Please remove this warning as the breaking change is about to happen.

Reply:
resolved